hosts/web02/postgresql.nix

```diff
@@ -0,0 +1,31 @@
+{
+  config,
+  inputs,
+  pkgs,
+  ...
+}:
+{
+  imports = [
+    inputs.self.nixosModules.backup
+  ];
+
+  services.postgresql = {
+    enable = true;
+    package = pkgs.postgresql_17;
+  };
+
+  services.postgresqlBackup = {
+    enable = true;
```

Comment:
These days I tend to make the postgresql backup in the borgbackup job.
This is so that the dump is fresh when the actual backup is happening.
This also makes monitoring a bit easier: https://github.com/NixOS/nixos-wiki-infra/blob/28750f7f2829ac134d2786675d7753512444ae00/modules/nixos-wiki/backup.nix#L131

Reply:
I'll probably change this to pgbackrest for hourly/daily/weekly backups, not really needed for grafana but I'll reuse it for buildbot, hydra db.

Reply:
I'll do pgbackrest in another PR, it requires a bit of configuration to setup it up correctly, I think daily backups of hydra db and grafana are sufficient for now.